.github/workflows/dependabot-pr.yml

```diff
@@ -43,6 +43,6 @@ jobs:
             git config user.email "49699333+dependabot[bot]@users.noreply.github.com"
             git config user.name "dependabot[bot]"
             git add **/Cargo.lock
-            git commit -am "update all Cargo files"
+            git commit -am "Update all Cargo files"
             git push
```

Comment:
i wonder `git push` works correctly here with `pull_request_target`.. just don't know enough about details of github actions.

Reply:
this one should work both `pull_request` and `pull_request_target` because we clone the repo by the PAT. however we don't use the `pull_reqeust_target` now!